docs/tutorials/08_cvar_optimization.ipynb

```diff
@@ -17,6 +17,8 @@
     "For a given set of shots with corresponding objective values of the considered optimization problem, the CVaR with confidence level $\\alpha \\in [0, 1]$ is defined as the average of the $\\alpha$ best shots.\n",
     "Thus, $\\alpha = 1$ corresponds to the standard expected value, while $\\alpha=0$ corresponds to the minimum of the given shots, and $\\alpha \\in (0, 1)$ is a tradeoff between focusing on better shots, but still applying some averaging to smoothen the optimization landscape.\n",
     "\n",
+    "Note that this note uses the former VQE class based on opflow because `CVaRExpection` is opflow. \n",
```

Comment:
I notice that `SamplingVQE` with `aggregaton` param allows CVaR optimization. I updated the note accordingly, but the result looks different from the former result. Alpha=0.25 does not converge to the optimal solution.

former
![image](https://user-images.githubusercontent.com/31178928/205025358-f48954a8-f254-4d9f-93c9-b718099950f8.png)
-------------------------

new
![image](https://user-images.githubusercontent.com/31178928/205025274-368be6c8-2dc9-42f3-8644-c51a66fdccdb.png)

Reply:
I discussed it with @Cryoris and he confirmed that there is a bug of SamplingVQE. He will fix it.

Reply:
I will try the fix https://github.com/Qiskit/qiskit-terra/pull/9221

Reply:
9221 looks good.
![image](https://user-images.githubusercontent.com/31178928/205054864-ffa4faba-7f81-4331-95e4-32366a9ef232.png)

Reply:
Ah ok, so for anything using CVar we ideally need a bug fix release of Terra that includes the above. Since CI here runs off Terra main as soon as the fix is merged there then the notebooks here will be fine. Which I guess means we can publish these new ones when 0.5.0 is released but for an end user that wants to do CVar we ideally need that fix in a official Terra bug fix release.